src/staging/package.json

```diff
@@ -46,6 +46,8 @@
     "react-icons": "^3.6.1",
     "react-material-ui-carousel": "^2.3.1",
     "react-reveal": "^1.2.2",
+    "react-router-dom": "^5.3.0",
+    "react-scrollspy": "^3.4.3",
```

Comment:
I believe @siva-ka had switched from react-scrollspy to react-use-scrollspy(pictured further below). There may have been a mix-up in branch merging lately. Do you explicitly use this library still?

Reply:
removed old scrollspy pkg